automation/test.sh

```diff
@@ -319,7 +319,7 @@ kubectl version
 
 mkdir -p "$ARTIFACTS_PATH"
 export KUBEVIRT_E2E_PARALLEL=true
-if [[ $TARGET =~ .*kind.* ]]; then
+if [[ $TARGET =~ .*kind.* ]] || [[ $TARGET =~ .*k3d.* ]]; then
```

Comment:
Could you please add a few words in the commit message about what this change does?

Reply:
added